src/pages/LinkList/components/LinksGrid/data/Links.ts

```diff
@@ -0,0 +1,772 @@
+export const Links = [
+  {
+    id: "106",
+    link: "https://afacebook.com",
+    status: "Await",
+    createdAt: new Date("12/22/2022"),
+    verifiedAt: null,
+  },
+  {
```

Comment:
Nice!

I think that you made a good decision, hardcoding it in this way.

I understand that the next step is using the services and React Query. 

Let me know if I can help you. 

Or if even, if you want, I can make honor to current behavior by refactoring it to use the dummy services and React Query. Since this code is small, I think that it would be quick and easy to understand.

Just let me know. Thanks!

Reply:
Thank you very much, we haven't thought about this yet, but I'll let you know.